### ISSUE TITLE
you can OR two Conds but can't AND them

### ISSUE THREAD

**Issue by brandly** [Sentenai/py-sentenai#59]
this is valid
```py
(myStream.val == 1) | (myStream.other == 2)
```

but this throws
```py
(myStream.val == 1) & (myStream.other == 2)
```

doesn't feel right

**Comment by xnomagichash** [comment]
agreed. even if it's not the preferred way it should be possible.

**Comment by brandly** [comment]
this is fixed in #81 (and you don't even need the parens)
